### PR TITLE
EPMRPP-38915: status change

### PR DESCRIPTION
**Pull request by Andrew-Omelchenko**

### CODE REVIEW

File: app/src/pages/inside/common/statusDropdown/statusDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }));
+  };
+
+  render() {
+    const {
+      data: { status },
+    } = this.props;
+    return (
+      <div className={cx('status-dropdown')}>
+        <InputDropdown
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]: Please try using `components/inputs/inputDropdownSorting` - it looks like the dropdown you need here

Comment (Andrew-Omelchenko) [reply app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]: Yes, `InputDropdownSorting` looks like dropdown I need here, but has another arrow shape and behavior, so when trying to use it I should modify not only styles, but also code inside this component.

Comment (DiscoElevator) [reply app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]: Could you give some examples?

Comment (Andrew-Omelchenko) [reply app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]:
```
        
![Screen Shot 2019-08-02 at 12 52 54 PM](https://user-images.githubusercontent.com/34836029/62361874-8d903880-b524-11e9-84fe-ef9699b2885f.png)
<InputDropdownSorting
          value={status}
          options={this.generateOptions(status)}
          onChange={this.updateItem}
          disabled={status === IN_PROGRESS}
          mobileDisabled
          transparent
        />
```

Comment (Andrew-Omelchenko) [reply app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]: Screen Shot 2019-08-02 at 12 52 54 PM

Comment (Andrew-Omelchenko) [reply app/src/pages/inside/common/statusDropdown/statusDropdown.jsx]: Screen Shot 2019-08-02 at 12 57 24 PM

**Pull request merged**